docs/test-configuration-options.md

```diff
@@ -0,0 +1,884 @@
+
+The default version of ChromeDriver when no value is specified depends on the version of Chrome used. See [ChromeDriver Versions and Downloads](http://chromedriver.chromium.org/downloads/version-selection) for more information.
+
+__Value Type__: string.
+
+__Example__: `"chromedriverVersion": "89"`
+<br/>
+
+### `geckodriverVersion`   
+
+__Description__: allows you to specify the Firefox GeckoDriver version. For Firefox version 80 and above, geckodriver defaults to latest driver version 0.27.0 when no version is specified.
+
+For a list of geckodriver versions, see [geckodriver Supported Platforms](https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html).
+
+__Value Type__: string.
+
+__Example__: "geckodriverVersion": "0.27.0"
+<br/>
+
+### `iedriverVersion`
+
+__Description__: allows you to specify the Internet Explorer Driver version. If no version is specified, it defaults to 2.53.1.
+
+The versions of Internet Explorer Driver we have available correspond to major Selenium releases - we do not have all the minor point releases (e.g., 3.12.0.4) available.
+
+We recommend setting the Selenium Version (see above) to correspond with the Internet Explorer Driver version you select.
+
+Sauce Labs supports launching 64-bit IE on our 64-bit VMs: Windows 7, Windows 8, and Windows 8.1. This provides a workaround for two known Selenium issues:
+* Using a 32 bit driver on a 64 bit operating system causes Selenium's screenshot feature to only capture the part of the page currently visible in the browser viewport [Selenium Issue 5876](https://code.google.com/p/selenium/issues/detail?id=5876).
+* Using a 64 bit driver on a 64 bit operating system causes text entry to be extremely slow [Selenium Issue 5516](https://code.google.com/p/selenium/issues/detail?id=5116).
+
+<details><summary><strong>Supported Internet Explorer Driver versions</strong></summary>
+
+*   `x64_2.29.0`
+*   `x64_2.39.0`
+*   `x64_2.40.0`
+*   `x64_2.41.0`
+*   `x64_2.42.0`
+*   `x64_2.43.0`
+*   `x64_2.44.0`
+*   `x64_2.45.0`
+*   `x64_2.46.0`
+*   `x64_2.48.0`
+*   `2.21.1`
+*   `2.21.2`
+*   `2.24.0`
+*   `2.25.3`
+*   `2.26.0`
+*   `2.28.0`
+*   `2.29.0`
+*   `2.30.1`
+*   `2.31.0`
+*   `2.32.2`
+*   `2.33.0`
+*   `2.34.0`
+*   `2.35.0`
+*   `2.35.1`
+*   `2.35.2`
+*   `2.35.3`
+*   `2.36.0`
+*   `2.37.0`
+*   `2.38.0`
+*   `2.39.0`
+*   `2.40.0`
+*   `2.41.0`
+*   `2.42.0`
+*   `2.43.0`
+*   `2.44.0`
+*   `2.45.0`
+*   `2.46.0`
+*   `2.48.0`
+*   `2.49.0`
+*   `2.50.0`
+*   `2.51.0`
+*   `2.52.0`
+*   `2.52.1`
+*   `2.52.2`
+*   `2.53.0`
+*   `2.53.1`
+*   `3.0.0`
+*   `3.1.0`
+*   `3.2.0`
+*   `3.3.0`
+*   `3.3.1`
+*   `3.4.0`
+*   `3.5.0`
+*   `3.5.1`
+*   `3.6.0`
+*   `3.7.0`
+*   `3.8.0`
+*   `3.9.0`
+*   `3.10.0`
+*   `3.11.0`
+*   `3.11.1`
+*   `3.12.0`
+*   `3.13.0`
+*   `3.14.0`
+*   `3.141.0`
+*   `3.150.1`
+
+</details>
+<br/>
+
+__Value Type__: string.
+
+__Example__: `"iedriverVersion": "3.141.0"`
+<br/>
+
+### `seleniumVersion`
```

Comment:
@titusfortner from Matt Dunn:

I think this whole section (`seleniumVersion`) needs a revamp. For example, the selenium version is ignored if using Chrome or Edge, as we don't use a selenium server at all. That's mentioned in a footnote here, but it's not very clear when looking at the working and table on the right.

Reply:
Yeah, I got stalled out yesterday revamping this section (it's a beast). Maybe I'll save it for last to maintain momentum. :)